arch-install.bash

```diff
@@ -182,14 +181,8 @@ else
       ;;
   esac
 
-
   mount "$root_fs" /mnt
 
-  if [[ $bios == 'uefi' ]]; then
-    mkdir /mnt/boot
-    mount "$efi_fs" /mnt/boot
-    mkdir -p /mnt/boot/efi
```

Comment:
I was only referring to `mkdir -p /mnt/boot/efi`. The other lines can stay, otherwise it won't mount the boot partition.

Reply:
You are right. Had a brainlag there